phpunit.xml.dist

```diff
@@ -1,33 +1,32 @@
-<!-- 
+<!--
 	PHPUnit configuration for SilverStripe
-
-	Requires PHPUnit 3.5+
-
-	Usage: 
+	Requires PHPUnit 5+
+	Usage:
 	 - "phpunit": Runs all tests in all folders
 	 - "phpunit framework/tests/": Run all tests of the framework module
 	 - "phpunit framework/tests/filesystem": Run all filesystem tests within the framework module
 	 - "phpunit framework/tests/filesystem/FolderTest.php": Run a single test
 	 - "phpunit <dash><dash>coverage-html assets/": Generate coverage report (replace <dash> with "-", requires xdebug)
-	
 	More information:
 	- http://www.phpunit.de/manual/current/en/textui.html
 	- http://doc.silverstripe.org/framework/en/topics/testing/#configuration
-
 	It is safe to remove this file for normal website operation.
 -->
 <phpunit bootstrap="framework/tests/bootstrap.php" colors="true">
 
 	<testsuite name="Default">
 		<directory>mysite/tests</directory>
-		<directory>cms/tests</directory>
-		<directory>framework/tests</directory>
+		<directory>vendor/cwp/cwp-core/tests</directory>
+		<directory>vendor/silverstripe/admin/tests/php</directory>
+		<directory>vendor/silverstripe/assets/tests/php</directory>
+		<directory>vendor/silverstripe/auditor/tests</directory>
+		<directory>vendor/silverstripe/config/tests</directory>
+		<directory>vendor/silverstripe/environmentcheck/tests</directory>
+		<directory>vendor/silverstripe/framework/tests/php</directory>
+		<directory>vendor/silverstripe/hybridsessions/tests</directory>
+		<directory>vendor/silverstripe/versioned/tests/php</directory>
```

Comment:
Some more granularity would be nice - see the examples which exist already: https://github.com/silverstripe/silverstripe-installer/blob/4.0.2/phpunit.xml.dist#L21-L43

We can plug them altogether in CI later on via `vendor/bin/phpunit --testsuite core,admin,blog,cwp,translations,whatever` for example